x11-libs/libva-intel-media-driver/files/libva-intel-media-driver-20.4.5_tesing_in_src_test.patch

```diff
@@ -0,0 +1,46 @@
+Run tests in src_test() instead of src_compile() and src_install()
```

Comment:
This is hopefully not too controversial and upstreamable

Reply:
I took the upstream testing and converted it from `custom build command` into `test`.
The test itself is untouched.

Reply:
Exactly! My hope is it can be upstreamed so you do not need to rebase it in future

Reply:
Unfortunately I have some doubts about accept of it by the upstream. They are using CI without additional "test" step, my changes require the additional step (as portage requires it). Not sure whether Intel will change their CI to do the things properly (from our point of view :) ).